xls/modules/rle/BUILD

```diff
@@ -97,6 +101,35 @@ xls_ir_verilog(
     verilog_file = "rle_enc.v",
 )
 
+verilog_library(
+    name = "rle_enc_verilog_lib",
+    srcs = [
+	":rle_enc.v",
+    ],
+)
+
+synthesize_rtl(
+    name = "rle_enc_synth",
+    top_module = "rle_enc",
+    deps = [
+        ":rle_enc_verilog_lib",
+    ],
+)
+
+place_and_route(
+    name = "rle_enc_place_and_route",
+    clock_period = "1",
+    core_padding_microns = 30,
+    placement_density = "0.8",
+    synthesized_rtl = ":rle_enc_synth",
+    target_die_utilization_percentage = "30",
+)
+
+gds_write(
+	name = "rle_enc_asic",
```

Comment:
can we replace `_asic` with `_gds` or `_layout`?

Reply:
changed to `rle_enc_gds`